src/test/kotlin/uk/gov/dluhc/printapi/rest/aed/UpdateAnonymousElectorDocumentByApplicationIdIntegrationTest.kt

```diff
@@ -92,4 +102,151 @@ internal class UpdateAnonymousElectorDocumentByApplicationIdIntegrationTest : In
             .hasError("Not Found")
             .hasMessage("Certificate for eroId = $ERO_ID with sourceType = ANONYMOUS_ELECTOR_DOCUMENT and sourceReference = $unknownSourceReference not found")
     }
+
+    @Test
+    fun `should update elector's email and phone number`() {
+        // Given
+        val eroResponse = buildElectoralRegistrationOfficeResponse(
+            id = ERO_ID,
+            localAuthorities = listOf(buildLocalAuthorityResponse(gssCode = GSS_CODE), buildLocalAuthorityResponse())
+        )
+        wireMockService.stubCognitoJwtIssuerResponse()
+        wireMockService.stubEroManagementGetEroByEroId(eroResponse, ERO_ID)
+        val originalEmailAddress = aValidEmailAddress()
+        val originalPhoneNumber = aValidPhoneNumber()
+        val aed = buildAnonymousElectorDocument(
+            gssCode = GSS_CODE,
+            sourceReference = APPLICATION_ID,
+            contactDetails = buildAedContactDetails(email = originalEmailAddress, phoneNumber = originalPhoneNumber)
+        )
+        anonymousElectorDocumentRepository.save(aed)
+        Thread.sleep(2000)
```

Comment:
Can we not thread sleep this for just a second (same comment for other tests in this class)